warden/group/manager_sql.go

```diff
@@ -88,58 +94,56 @@ func (m *SQLManager) GetGroup(id string) (*Group, error) {
 	}, nil
 }
 
-func (m *SQLManager) DeleteGroup(id string) error {
-	if _, err := m.DB.Exec(m.DB.Rebind("DELETE FROM hydra_warden_group WHERE id=?"), id); err != nil {
-		return errors.WithStack(err)
+func (m *SQLManager) deleteGroup(id string) func(tx *sqlx.Tx) error {
+	return func(tx *sqlx.Tx) error {
+		if _, err := tx.Exec(m.DB.Rebind("DELETE FROM hydra_warden_group WHERE id=?"), id); err != nil {
+			return err
```

Comment:
Normal errors do not have stack traces in go, to add a stack trace do `return errors.WithStack(err)`